src/main/webapp/profile-page-test.html

```diff
@@ -0,0 +1,34 @@
+<!-- TODO: Replace this page with the real profile page. -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Temporary Profile Page</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>   
+  </head>
+  <body onload="getUserData(); fetchBlobstoreUrl();">
```

Comment:
newline

Reply:
also i didn't know you could do this I just create a single that calls both and then pass that in :0 very cool